test/benchee_test.exs

```diff
@@ -253,6 +253,40 @@ defmodule BencheeTest do
     end
   end
 
+  test ".run accepts atom keys for jobs" do
+    capture_io fn ->
```

Comment:
Are you using `capture_io` here to just turn off all IO for this test so it doesn't clog up the test output?

If so, I think that this might be a good feature to propose to add to `ExUnit`. I'm sure anyone working on any CLI applications would like to have this as an available configuration option, don't you?